### PR TITLE
Add meetup routes & populate homepage slider issue #9

### PR DESCRIPTION
**Pull request by TalhaKhatri**
-Add meetup routes
-Create meetup service
-Fetch meetups from service
-Populate homepage slider

@mashhoodr Up for review

### CODE REVIEW

**Comment by mashhoodr** [comment]
@afirdousi can you please review

File: server/models/meetup.model.js [hunk 1]
```diff
@@ -36,9 +36,9 @@ var meetupSchema = Schema({
       ]
     }
   ],
-  subscriber:[
+  subscribers:[
     {
-      userID: ObjectId,
+      userID: String,
```
Comment (afirdousi) [inline server/models/meetup.model.js]: Why would you use type String for userID?

If we need the user model within meetup model, we should do something along the lines of 


`
userID: {
        type:ObjectId ,
        ref:'Users'
    }
`

**Comment by afirdousi** [comment]
@TalhaKhatri what do you think of the above suggestion? Can we make the adjustments, rebase with master and raise PR again?

Thanks.

File: server/models/meetup.model.js (hunk 1 above)
Comment (TalhaKhatri) [reply server/models/meetup.model.js]: @afirdousi Ah yes, will do that. I probably left that by accident while trying something out.

**Comment by TalhaKhatri** [comment]
@afirdousi Branch has been rebased and adjustments have been made. Please review, thanks!

**Pull request merged**